docs/references/phasing.tex

```diff
@@ -1,111 +1,179 @@
+It is important to note that there is a discrepancy between \texttt{novas}-generated coordinates and those generated by the other two packages, of order 100--200 {\textmu}as. This difference is generally small enough to be insignificant for most application, and when using LSTs calculated by \texttt{novas} the difference in calculated position (in terms of horizontal coordinates) reduces by an order of magnitude. To that end, it is often best practice to use the same package for both calculating the LSTs and for calculating astrometric positions.
 
-\section{Testing \texttt{pyuvdata}'s phasing}
+\subsection{Rotation to $uvw$}\label{ssec:rotation}
+Once the astrometry calculations are complete, there are two potential paths toward calculating the $uvw$-coordinates, though we'll focus on the more typical path, which uses the antenna positions, $\mathbf{r}_{\rm ant}$ (as recorded in \texttt{UVData.antenna\_positions})\footnote{The alternative path works with the previously calculated \texttt{UVData.uvw\_array}, with the previously applied rotations removed, such that the projected baseline positions are measured relative to the line connecting 0$^\circ$ latitude and longitude and the Earth center, after which the rotations mentioned below are applied. Thanks to the commutative properties of matrices, this is mathematically equivalent to rotating the antenna positions first before differencing them to get the relative baseline positions.}. In total, we have three rotations to perform. We will denote these as $\mathcal{R}_{a,b}$, which is the rotation matrix about axis $a$ by amount $b$. Inside of a `UVData` object, the antenna positions are recorded in ECEF coordinates (or the lunar equivalent), which means that the array is oriented along the line that connects the center of the Earth to the position at 0$^\circ$ latitude and longitude, where we'll refer to the plane that spans this direction and the northern pole as the Greenwich meridian. Therefore, in order to calculate the projected baselines, one needs to know the Greenwich Hour Angle (GHA) -- the separation between the Greenwich meridian and the meridian that goes through the north pole and the phase center position. This is relatively easy to calculate as:
```

Comment:
`UVData` -> \texttt{UVData}

Reply:
"the antenna positions are recorded in ECEF coordinates". Maybe: the antenna positions are recorded in ECEF coordinates relative to the \texttt{UVData.telescope_location}

Reply:
Fixed.